pkg/gui/confirmation_panel.go

```diff
@@ -15,25 +15,27 @@ import (
 
 func (gui *Gui) wrappedConfirmationFunction(function func(*gocui.Gui, *gocui.View) error) func(*gocui.Gui, *gocui.View) error {
 	return func(g *gocui.Gui, v *gocui.View) error {
+		if err := gui.closeConfirmationPrompt(); err != nil {
```

Comment:
following lazygit's lead by always returning focus BEFORE running a handler, to prevent all kinds of shenanigans